source/blog/2017-08-16-ovirt-ansible-roles-how-to-use.html.md

```diff
@@ -0,0 +1,153 @@
+---
+title: How to use oVirt Ansible roles
+author: omachace
+tags: Ansible, Roles, Automation, Configuration management
+date: 2017-08-16 15:00:00 CET
+comments: true
+published: true
+---
+
+The recent post, [An Introduction to Ansible Roles](./2017-07-19-ovirt-ansible-roles-an-introduction.html.md), covered the new roles introduced in the oVirt 4.1.6 release. This follow-up post will explain how to set up and use Ansible roles either using Ansible Galaxy or oVirt Ansible Roles RPM.
+
+## Ansible Galaxy
+
+[Ansible Galaxy](https://galaxy.ansible.com/) stores many different Ansible roles, making your life simpler. oVirt Ansible roles are also stored in Galaxy. To install those roles from Ansible Galaxy, perform the following steps:
```

Comment:
Please change: 
"To make life easier, Ansible Galaxy stores multiple Ansible roles, including oVirt roles. To install the roles, perform the next steps:"